pages/examples/forms/non-interactive-content/README.md

```diff
@@ -66,7 +66,7 @@ When navigating using the `Tab` key, the easiest way to prevent non-focusable co
 
 While this may be tempting, it is very bad style: only elements that provide some interaction (buttons, links, form controls, etc.) should be focusable. Otherwise keyboard only users may be confused, as they may think that (for example) a paragraph can be interacted with, although it cannot.
 
-Also, it is unfortunate that the descriptive text of a form control always is announced to screen readers **after** interacting with it (right after leaving it using `Tab`).
+Also, it is unfortunate that the descriptive text of a form control always is announced to screen readers **after** interacting with it (right after leaving it using tap or `Tab`).
```

Comment:
I think it's not important how the item is left; I think rephrasing it like the following is more generic:

> Also, it is unfortunate that this way the descriptive text of a form control is announced to screen readers **after** perceiving (and possibly interacting with) it.